h2o-docs/src/booklets/v2_2015/source/sw/sections/pysparkling.tex

```diff
@@ -72,20 +72,20 @@ \subsection{Getting Started:}
 Then create an H2O cloud inside the Spark cluster and import H2O-Python package:
 \begin{lstlisting}[style=Scala]
 from pysparkling import *
-hc= H2OContext(sc).start()
+hc = H2OContext(spark).start()
```

Comment:
This should be `hc = H2OContext.getOrCreate(spark)` Sorry, forgot about this in the initial review. Thanks for all the updates @angela0xdata !